zerver/openapi/zulip.yaml

```diff
@@ -14507,6 +14507,50 @@ paths:
         given their user ID.
       parameters:
         - $ref: "#/components/parameters/UserId"
+        - name: is_spammer
+          in: query
+          description: |
+            Whether or not the user should be marked as a spammer.
+
+            **Changes**: New in Zulip 11.0 (feature level ZF-01a225).
+          content:
+            application/json:
+              schema:
+                type: boolean
+              example: true
+          required: false
+        - name: message_delete_action
```

Comment:
Should we consider calling this just `action`, to provide the opportunity to extend later to actions that aren't specifically deleting messages?